Scripts/Runtime/Entities/Tasks/AbstractTaskSystem.cs

```diff
@@ -9,26 +15,33 @@ namespace Anvil.Unity.DOTS.Entities
     public abstract partial class AbstractTaskSystem : AbstractAnvilSystemBase,
                                                        ITaskSystem
     {
+        private static readonly Type I_PROXY_DATA_STREAM_TYPE = typeof(IProxyDataStream);
```

Comment:
I'm not sure it's beneficial to do this. I think this is just a cached retrieval of the Type instance anyway.
It would be cleaner to just do this at the start of `CreateProxyDataStreams()`

Reply:
huh neat, When I was looking it up `typeof` is frequently labelled as "compile-time" resolved which isn't really accurate but MSDN does repeatedly say it is an efficient implementation.

That was a good post you linked. I agree with the guy saying it is faster but you shouldn't use it unless you're solving a provable bottleneck or need that type repeatedly across many scopes.